u-root.go

```diff
@@ -84,6 +90,9 @@ func init() {
 	statsOutputPath = flag.String("stats-output-path", "", "Write build stats to this file (JSON)")
 
 	statsLabel = flag.String("stats-label", "", "Use this statsLabel when writing stats")
+
+	version = flag.String("version", "@v1.0.0", "Version to use when modules are enabled")
```

Comment:
I find this quite confusing, since most tools would print their version, not have one specified for a specific purpose.
If need be, I feel like an env var would be more natural, similar to the other build settings like `GOARCH` and `GO111MODULE`.